pkg/csi/cinder/openstack/openstack.go

```diff
@@ -60,6 +61,11 @@ type IOpenStack interface {
 	DeleteSnapshot(snapID string) error
 	GetSnapshotByID(snapshotID string) (*snapshots.Snapshot, error)
 	WaitSnapshotReady(snapshotID string) error
+	CreateBackup(name, volID string, snapshotID string, tags *map[string]string) (*backups.Backup, error)
```

Comment:
it there a reason to pass a pointer to the `map[string]string`?

Reply:
I couldn't find the reason why they did it this way in `CreateSnapshot()`, so I just did the same. If I undestood the reason I would be more comfortable changing it :sweat_smile:

Reply:
I'd change the `tags *map[string]string` to `tags map[string]string` in all methods, since there is no need to pass the pointer. https://play.golang.com/p/-NjjexjOEx5